src/components/layout/header-actions.tsx

```
@@ -0,0 +1,77 @@
+          <Link
+            href={linkPath as string}
+            className="size-8 rounded-full border grid place-content-center"
+          >
+            <Icon className="size-4 text-muted-foreground" />
+          </Link>
+        ) : (
+          <div
+            role="button"
+            onClick={buttonAction}
+            className="size-8 rounded-full border grid place-content-center"
+          >
```

Biome (2.1.2)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)